debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -507,7 +507,14 @@ private SourceConnector instantiateConnector(final String connectorClassName) th
 
     private Map<String, String> getConnectorConfig(final SourceConnector connector, final String connectorClassName) throws EmbeddedEngineRuntimeException {
         Map<String, String> connectorConfig = workerConfig.originalsStrings();
-        Config validatedConnectorConfig = connector.validate(connectorConfig);
+        Config validatedConnectorConfig = null;
+        try {
+            validatedConnectorConfig = connector.validate(connectorConfig);
+        }
+        catch (java.lang.NoClassDefFoundError ex) {
```

Comment:
I'd catch general `Exception` as `validate()` can throws also other errors like `ConnectException`.

Reply:
thanks, thanks, I've expanded the scope, was that what you meant?

Reply:
yes, thanks for quick response